docs/pages/branding/about.tsx

```diff
@@ -192,8 +373,9 @@ function BrandingSupportUs() {
               Evangelize Material-UI by linking to{' '}
               <Link href="https://material-ui.com">material-ui.com</Link> on your website, every
               backlink matters! Follow us on{' '}
-              <Link href="https://twitter.com/MaterialUI">Twitter</Link>, like and retweet the
-              important news. Or just talk about us with your friends.
+              <Link href="https://twitter.com/MaterialUI">Twitter</Link>, retweet the important
+              news. <Link href="https://github.com/mui-org/material-ui">Star us</Link> on GitHub
+              ⭐️. Or just talk about us with your friends.
```

Comment:
Always thought that was a bit sad. 🤓 
```suggestion
              ⭐️. Or share what you've built with Material-UI.
```